Resources/public/mapbender.element.digitizer.js

```diff
@@ -1723,16 +1726,21 @@
                                 });
                                 return;
                             }
-                            _.extend(dataItem, response.dataItem);
+                            _.extend(dataItem, response);
+                            schema.options[dataItem[uniqueKey]] = dataItem[textKey];
                             if(isNew) {
-                                var textKey = item.dataStore.text;
-                                var uniqueKey = item.dataStore.uniqueId;
```

Comment:
Removed redundant init of uniqueKey and moved textKey to the top of the function for better codestyle